src/pages/docs/infrastructure/workers/autoscaling-workers/cluster-configuration.md

```diff
@@ -10,25 +10,30 @@ navOrder: 20
 The Kubernetes worker has been proven to be effective on a variety of installations.
 But some configurations are more complex than others!
 
-There's three factors which affect the effort the likelihood of success:
+There's three factors which affect the likelihood of success:
 1. Kubernetes distribution (eg AKS, EKS, GKE ...)
```

Comment:
Should we say 'distributions and managed services'? Formally speaking, AKS is not a distribution.